README.md

```diff
@@ -245,7 +245,7 @@ Each generic operation has a synchronous and asynchronous version.
 ```csharp
 // Generic read from a path with query parameters
 var readPath = "/some/path"
-Dictionary<string, object> queryParams = new Dictionary<string, object>
+IDictionary<string, object> queryParams = new Dictionary<string, object>
```

Comment:
I'd like to keep the examples as simple as possible with the concrete implementation, but I suppose it shows that the methods accept an `IDictionary` now, which makes sense

Reply:
Yes, I figured that's the typical way you would indicate it but I'm OK using concrete dictionary or even in-lining this param for the example.